blockchains/ethereum/assets/0xEEF9f339514298C6A857EfCfC1A762aF84438dEE/info.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Hermez Network Token",
+    "website": "https://hermez.io/",
+    "source_code": "https://github.com/hermeznetwork",
+    "short_description": "Hermez is a decentralized zk-rollup focused on scaling payments and token transfers on top of Ethereum.",
+    "socials": [
+        {
+            "name": "Twitter",
+            "url": "https://twitter.com/hermez_network",
+            "handle": "hermez_network"
+        },
+        {
+            "name": "Telegram",
+            "url": "https://t.me/hermez_network",
+            "handle": "hermez_network"
+        },
+	{
+            "name": "GitHub",
+            "url": "https://github.com/hermeznetwork",
+            "handle": "hermeznetwork"
+        }
+    ],
+    "explorer": "https://etherscan.io/address/0xeef9f339514298c6a857efcfc1a762af84438dee"
```

Comment:
Please update it to `https://etherscan.io/token/0xeef9f339514298c6a857efcfc1a762af84438dee`